frontend/src/Components/Home/PopularArticles/ArticleImage/ArticleImage.scss

```diff
@@ -4,14 +4,11 @@
 @import "bootstrap/scss/grid";
 @import "bootstrap/scss/mixins";
 
-.article-img {
-  @include media-breakpoint-down(xs) {
-    width: 200px;
+.popular-article-img {
+  @include media-breakpoint-down(sm) {
+    width: 220px;
   }
-  @include media-breakpoint-up(sm) {
-    width: 400px;
-  }
-  @include media-breakpoint-up(lg) {
+  @include media-breakpoint-up(md) {
     width: 250px;
   }
   @include media-breakpoint-up(xxl) {
```

Comment:
There is a gap between sm and md now where the popular image width is not accounted for so the image is very big at like a screen width around 800px, address this gap.